python/metatensor-torch/metatensor/torch/atomistic/documentation.py

```diff
@@ -0,0 +1,293 @@
+from typing import Dict, List, Optional
+
+import torch
+
+from ..documentation import Labels, TensorBlock
+
+
+class System:
```

Comment:
Do we also want to provide converters as we do it in rascaline: i.e from a ase object to an systems object?

Reply:
Okay, 100% okay for me. Once we replaced the system in rascaline we could add a comment here that some converters exist in the rascaline or the `metatensor-models` package.